### DISCUSSION TITLE
Can't access Nextcloud after update: Update needed Please use the command line updater because updating via browser is disabled in your config.php.

### DISCUSSION THREAD

**Discussion by cordvision** [nextcloud/all-in-one#5189] [Questions] [answered]
First of all, I’m a complete beginner so please explain it like I’m 5 years old.

My set-up:
I installed Nextcloud on my Asustor NAS through the built in App Store (AppCentral). I believe this means it installed a docker container. I can access the console through another app that I also installed in the AppCentral that is called Portainer CE.

It worked flawlessly until today:
I updated Nextcloud through the AppCentral of my Asustor NAS (version 29.0.4.r02
). Since then, I’m getting the following message as I’m trying to access my Nextcloud:

_Update needed
Please use the command line updater because updating via browser is disabled in your config.php._

I tried to follow various suggestions found online but with no success. Could somebody provide me simple step by step instructions on how I can get NC back up running? Any help is greatly apprechiated!

**Comment by szaimen** [comment, marked as answer]
Hi, it looks like you are not using Nextcloud AIO?
In this case, please open a new thread for this at https://help.nextcloud.com

**Reply by cordvision** [reply]
Oh sorry, you are right... my bad.
